src/pages/Checkout/CheckoutPage.vue

```diff
@@ -317,6 +316,9 @@ export default {
 				{ __typename: 'Credit', creditType: 'redemption_code' }
 			);
 			this.hasFreeCredits = _get(data, 'shop.basket.hasFreeCredits');
+			if (this.kivaCards || this.redemption_credits || this.hasFreeCredits) {
```

Comment:
We should only check for `if (this.redemption_credits || this.hasFreeCredits) {` because we still want to allow guest users to purchase kiva cards. It's using kiva cards (redemption credit) that we want to block.